data/use-get-groups.ts

```diff
@@ -106,7 +107,7 @@ export const useGetGroup = (groupId: string): GetGroupResult => {
     }
   );
   const { data: rolesData, error: rolesError } = useSWR(
-    `/management/roles`,
+    `/management/tenant/${selectedTenant?.rec_id || null}/roles`,
```

Comment:
```suggestion
    selectedTenant ? `/management/tenant/${selectedTenant.rec_id}/roles` : null,
```

using `/management/tenant/${selectedTenant?.rec_id || null}/roles` will cause fetching to this url `/management/tenant/null/roles` when no tenant is selected.
I assume that this intended for not fetching when no tenant is selected, so it requires the key for useSWR to be `null`